spec/model/post_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```

Comment:
- [ ] All models should be tested. Kindly add tests for the Like and Comment models.